Makefile

```diff
@@ -16,6 +16,12 @@ default: exe
 exe:
 	dune build bin/ocamlformat.exe
 
+.PHONY: install
+install:
+	dune build @install
+	dune install --prefix _install
+	dune install
```

Comment:
Is this expected to run install twice?

Reply:
I also had this question, seems like the second `dune install` should be removed.